package.json

```diff
@@ -247,6 +247,18 @@
           "when": "view == queryHistory",
           "title": "%mssql.clearAllQueryHistory%",
           "group": "secondary"
+        },
+        {
+          "command": "mssql.objectExplorer.enableGroupBySchema",
+          "when": "view == objectExplorer && !config.mssql.objectExplorer.groupBySchema",
+          "title": "%mssql.objectExplorer.enableGroupBySchema%",
+          "group": "secondary"
```

Comment:
For VS Code MSSQL seems like it'd be better to have it as a navigation item so it shows up on the title bar. That's a lot easier to see and work with - the secondary menu is still a bit too hidden IMO. 

@erinstellato-ms thoughts on this?

Reply:
IMO I don't think people will toggle this setting a lot. Therefore, I am not sure about making it visible all the time.  They will either stick with schema tree or regular tree. We can enable this setting by default in the next version and let users disable it if they want to.

Reply:
It's not like there isn't plenty of room there. And I just think that hiding it in the secondary menu means that most people aren't going to know it exists unless they're really diligent about reading the release notes. 

Just seems a shame to put all the work into this feature and then make it difficult for people to discover it. We can always hide the action later if people don't like it there.